packages/core/src/components/dropdown/test/default/dropdown.e2e.ts

```diff
@@ -0,0 +1,82 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/dropdown/test/default/index.html';
+
+test.describe('tds-dropdown-default', () => {
+  test('renders default dropdown correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const dropdown = page.locator('tds-dropdown');
+    expect(dropdown).toBeTruthy();
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('default dropdown have label outside', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const dropdown = page.locator('tds-dropdown');
+    const labelOutside = dropdown.locator('div[class="label-outside"]');
```

Comment:
This test still passed when i changed to label inside 💡 Something's not working

Reply:
However, I think just having the screenshot could be good enough in this case as this is testing implementation details 🤔

Reply:
Kinda updated this I think, The playwright locator still tries to find the label-outside even tho you put it for label inside. So i just now check if the text contains withinit and it breaks if you change to label inside.